python/lsst/analysis/tools/analysisParts/shapeSizeFractional.py

```diff
@@ -0,0 +1,177 @@
+from __future__ import annotations
+
+import numpy as np
+
+from typing import cast
+from lsst.analysis.tools.vectorActions.selectors import VectorSelector
+from lsst.analysis.tools.vectorActions.vectorActions import LoadVector, MagColumnNanoJansky
+
+from lsst.pex.config import Field
+from lsst.pipe.tasks.configurableActions import ConfigurableActionField, ConfigurableActionStructField
+
+from ..interfaces import (
+    AnalysisAction,
+    KeyedData,
+    KeyedDataAction,
+    Scalar,
+    ScalarAction,
+    VectorAction,
+    KeyedDataSchema,
+    Vector,
+)
+from ..scalarActions import CountAction, MedianAction, SigmaMadAction
+from ..keyedDataActions import AddComputedVector, KeyedScalars
+from ..vectorActions import SnSelector, StellarSelector, FractionalDifference
+from ..vectorActions.calcShapeSize import CalcShapeSize
+
+
+class _AproxMedian(ScalarAction):
```

Comment:
Should this class be moved to scalarActions/scalarActions.py ?

Reply:
Also, is "Aprox" here meant to be short for "Approximately"?  If so, check your "p"s (and if not, perhaps a full spell-out would avoid confusion?)

Reply:
This was intended to be something specific implementation detail specific for how this calculates something, and may get replaces with something better later. I have no strong feeling if others really want to move it I will not argue, but if others don't have strong feelings I would prefer to leave it where it is.